crates/viewer/re_data_ui/src/blob.rs

```diff
@@ -142,3 +157,46 @@ pub fn blob_preview_and_save_ui(
         });
     }
 }
+
+fn show_video_blob_info(
+    ui: &mut egui::Ui,
+    ui_layout: UiLayout,
+    video_result: &Result<re_renderer::video::Video, re_renderer::video::VideoError>,
+) {
+    match video_result {
+        Ok(video) => {
+            if ui_layout.is_single_line() {
+                return;
+            }
+
+            re_ui::list_item::list_item_scope(ui, "video_blob_info", |ui| {
+                ui.list_item_flat_noninteractive(re_ui::list_item::LabelContent::new(
+                    "Video properties",
+                ));
+                ui.list_item_flat_noninteractive(
+                    PropertyContent::new("Dimensions").value_text(format!(
+                        "{}x{}",
+                        video.width(),
+                        video.height()
+                    )),
+                );
+                ui.list_item_flat_noninteractive(PropertyContent::new("Duration").value_text(
+                    format!(
+                        "{}",
+                        re_log_types::Duration::from_millis(video.duration_ms() as _)
+                    ),
+                ));
+                ui.list_item_flat_noninteractive(
+                    PropertyContent::new("Codec").value_text(video.codec()),
+                );
+            });
```

Comment:
It would be great with a mini-player of the video here!

Reply:
that would be awesome indeed! 😄 
But keep in mind that that requires creating a new decoder (decoders are stateful to where they are right now) which can get expensive quickly. Definitely would like to try this though in a future iteration!